### PR TITLE
Add JSON output for TPCH binary

### PR DESCRIPTION
**Pull request by AdamGS**
Adds a new `quick-benchmark` run that runs the tpch binary and outputs gh benchmark compatible JSON. For now its triggered by adding the `quick-benchmark` label to a PR. Current runtime is ~15 minutes instead of the criterion based variant that takes >30, and can probably be improved by reducing the number of default iterations/narrowing the amount of formats we actually benchmark.

### CODE REVIEW

File: .github/workflows/tpch_quick.yml [hunk 1]
```diff
@@ -0,0 +1,67 @@
+name: Quick Benchmark
```
Comment (robert3005) [inline .github/workflows/tpch_quick.yml]: Ideally this would replace existing criterion tpch benchmarks

**Review by robert3005 — APPROVED** [review]

File: bench-vortex/Cargo.toml [hunk 2]
```diff
@@ -46,6 +46,7 @@ mimalloc = { workspace = true }
 object_store = { workspace = true, features = ["aws"] }
 parquet = { workspace = true, features = ["async"] }
 prettytable-rs = { workspace = true }
```
Comment (robert3005) [inline bench-vortex/Cargo.toml:48]: would be nice to get rid of this in a followup

**Comment by a10y** [comment]
I'd even be happy to just get the `vortex-file-compressed` results tbh, most of the time when I'm seeing how much a change affects existing benchmarks that's all I look at anyway, since it's rare that we do anything that would change the arrow or parquet results.

**Comment by AdamGS** [comment]
@a10y the benefits for arrow/parquet is that it makes datafusion improvements clearer, and just keeps the graphs nice.

**Comment by a10y** [comment]
Yea that makes sense for graphs, I just meant for "quick" PR runs you usually only care about change in vortex performance

**Comment by AdamGS** [comment]
just replaced the benchmark run fully. You can run it locally with `--only_vortex` and that juts does arrow vs vortex and prints a nice table.

**Pull request merged**